quickstarts/microsoft.costmanagement/finops-hub/azuredeploy_china.json

```diff
@@ -0,0 +1,1492 @@
+      }
+    },
+    "parameters": {
+      "hubName": {
+        "type": "string",
+        "metadata": {
+          "description": "Optional. Name of the hub. Used to ensure unique resource names. Default: \"finops-hub\"."
+        }
+      },
+      "location": {
+        "type": "string",
+        "defaultValue": "[resourceGroup().location]",
+        "metadata": {
+          "description": "Optional. Azure location where all resources should be created. See https://aka.ms/azureregions. Default: Same as deployment."
+        }
+      },
+      "storageSku": {
+        "type": "string",
+        "defaultValue": "Premium_LRS",
+        "metadata": {
+          "description": "Optional. Storage SKU to use. LRS = Lowest cost, ZRS = High availability. Note Standard SKUs are not available for Data Lake gen2 storage. Allowed: Premium_LRS, Premium_ZRS. Default: Premium_LRS."
+        },
+        "allowedValues": [
+          "Premium_LRS",
+          "Premium_ZRS"
+        ]
+      },
+      "keyVaultSku": {
+        "type": "string",
+        "defaultValue": "standard",
+        "metadata": {
+          "description": "Optional. Allowed: standard, premium. Default: standard. Only standard is supported for Azure China"
+        },
+        "allowedValues": [
+          "standard",
+          "premium"
+        ]
+      },
+      "tags": {
+        "type": "object",
+        "defaultValue": {},
+        "metadata": {
+          "description": "Optional. Tags to apply to all resources. We will also add the cm-resource-parent tag for improved cost roll-ups in Cost Management."
+        }
+      },
+      "exportScopes": {
+        "type": "array",
+        "defaultValue": [],
+        "metadata": {
+          "description": "Optional. List of scope IDs to create exports for."
+        }
+      }
+    },
+    "resources": [
+      {
+        "type": "Microsoft.Resources/deployments",
+        "apiVersion": "2022-09-01",
+        "name": "hub",
+        "properties": {
+          "expressionEvaluationOptions": {
+            "scope": "inner"
+          },
+          "mode": "Incremental",
+          "parameters": {
+            "hubName": {
+              "value": "[parameters('hubName')]"
+            },
+            "location": {
+              "value": "[parameters('location')]"
+            },
+            "storageSku": {
+              "value": "[parameters('storageSku')]"
+            },
+            "keyVaultSku": {
+              "value": "[parameters('keyVaultSku')]"
+            },
+            "tags": {
+              "value": "[parameters('tags')]"
+            },
+            "exportScopes": {
+              "value": "[parameters('exportScopes')]"
+            }
+          },
+          "template": {
+            "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+            "contentVersion": "1.0.0.0",
+            "metadata": {
+              "_generator": {
+                "name": "bicep",
```

Comment:
given that this was originally authored in bicep, can we please include the bicep code in the PR?